cpp/box/Box.h

```diff
@@ -367,7 +367,7 @@ class Box
      *  \param masses Optional array of masses, of length Nvecs
      *  \return Center of mass as a vec3<float>
      */
-    vec3<float> centerOfMass(vec3<float>* vecs, size_t Nvecs, const float* masses) const
+    vec3<float> centerOfMass(vec3<float>* vecs, size_t Nvecs, const float* masses = nullptr) const
```

Comment:
The masses passed in from python are guaranteed to be non-null.

Reply:
The masses coming from `ClusterProperties::compute` are not.
https://github.com/glotzerlab/freud/blob/ba5998eb4f6a51ba6930e200066abb09390ab059/cpp/cluster/ClusterProperties.cc#L60

This causes `clang-tidy` to fail:
https://github.com/glotzerlab/freud/actions/runs/9649688151/job/26613682932#step:6:57